Installer/config/FanartHandler.Holidays.xml

```diff
@@ -373,24 +428,44 @@
 		<DayOfYear>256</DayOfYear>
 		<NameDE>Tag der Programmierer</NameDE>
 	</Holiday>
+	<Holiday name="Daylight Savings Time Begins" shortname="DaylightSavings|DaylightSavingsBegin">

```

Comment:
Isn't the current universal implementation of calculation based on computer settings suitable for you? In my opinion, it takes into account all options if the Locale in Windows is configured correctly.

Reply:
In the Australian summer, New South Wales, Victoria, South Australia, Tasmania and the ACT turn their clocks forward one hour to Daylight Saving Time (DST).
Daylight Saving Time begins at 2am (AEST) on the first Sunday in October and ends at 3am (Australian Eastern Daylight Time) on the first Sunday in April.
Daylight saving is not observed in Queensland, the Northern Territory or Western Australia.

I live in Queensland and wanted to be notified when the other states went into DST, so no my computer clock is not going to work.

Reply:
Maybe I should take this out and add it to FanartHandler.Holidays.Custom.xml.
If so, would it be better to add the manual AU DST commented out in this PR, or just add it to my personal copy?

Reply:
> Maybe I should take this out and add it to FanartHandler.Holidays.Custom.xml.
> If so, would it be better to add the manual AU DST commented out in this PR, or just add it to my personal copy?

It turns out yes, because This is not basic behavior, but your personal preferences.

Just transfer these two events to Custom and they will work, but don’t forget to remove the flag in the main file that DST does not work in Australia.
Those. if you have TZ installed without time change, then this event should not appear for you, but because... You registered it in Custom, then it will be visible.

Reply:
OK, I will remove it now and add it to my personal Custom.